lib/jit_preloader.rb

```diff
@@ -8,18 +8,17 @@
 require 'jit_preloader/active_record/relation'
 require 'jit_preloader/active_record/associations/collection_association'
 require 'jit_preloader/active_record/associations/singular_association'
-if Gem::Version.new(ActiveRecord::VERSION::STRING) < Gem::Version.new("5.2.2")
+if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new("6.0.0")
+  require 'jit_preloader/active_record/associations/preloader/association'
```

Comment:
Would it make more sense to continue calling this `ar6_association`, to be explicit about what version it's for?